backend/tests/integration/user_workflows/test_user_worflow.py

```diff
@@ -141,6 +145,14 @@ def __init__(self) -> None:
 
 
 class TestUserWorkflow01:
+    @pytest.fixture(scope="class", autouse=True)
+    def workflow_local(self, prefect_test_fixture) -> Generator[WorkflowLocalExecution, None, None]:
+        original = config.OVERRIDE.workflow
+        workflow = WorkflowLocalExecution()
+        config.OVERRIDE.workflow = workflow
+        yield workflow
+        config.OVERRIDE.workflow = original
+
```

Comment:
As discussed this should not be needed as we now have WorkflowLocalExecution as default workflow for our tests
https://github.com/opsmill/infrahub/pull/4699/files#diff-aacea3e16f9fecd307f1a11567ba7a1abb8a6007266aec3b89dd65454130b76cR382

Reply:
Good catch, updated